docs/en/setup/backend/backend-storage.md

```diff
@@ -72,6 +72,8 @@ storage:
     oapAnalyzer: ${SW_STORAGE_ES_OAP_ANALYZER:"{\"analyzer\":{\"oap_analyzer\":{\"type\":\"stop\"}}}"} # the oap analyzer.
     oapLogAnalyzer: ${SW_STORAGE_ES_OAP_LOG_ANALYZER:"{\"analyzer\":{\"oap_log_analyzer\":{\"type\":\"standard\"}}}"} # the oap log analyzer. It could be customized by the ES analyzer configuration to support more language log formats, such as Chinese log, Japanese log and etc.
     advanced: ${SW_STORAGE_ES_ADVANCED:""}
+    storageMode: ${SW_STORAGE_ES_MODE:FUNCTIONAL_AGGREGATION}#There are 2 mode for indices storage, which are FUNCTIONAL_AGGREGATION and NO_AGGREGATION. The Storage behavior using NO_AGGREGATION is same with the storage behavior before 8.5.0 version. And, the Storage behavior using FUNCTIONAL_AGGREGATION means the telemetry data, that generated by OAL or MAL, would be partitioned by the functions of the OAL or MAL.
```

Comment:
I think we could do permanent change rather than 2 modes. This new one is always better and recommended. Right?

Reply:
We users should expect the storage to be compatible, giving them time to migrate to the new scheme.

Reply:
Actually, this wouldn't help. Because OAP can't write data in 2 kinds of storage implementations. At any point the switch happens, data lost forever.

Reply:
And you will make end users confused about choose what solution is better. They don't read codes.

Reply:
Or we can add some notice comments here to show the new schema is better than now. Or remove history storage mode in 9.0.0.